src/ResourceManager/Network/Commands.Network/ExpressRouteCrossConnection/CrossConnectionPeering/AzureRMExpressRouteCrossConnectionPeeringBase.cs

```diff
@@ -0,0 +1,140 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.Network
+{
+    using System.Collections.Generic;
+    using System.ComponentModel;
+
+    using Microsoft.Azure.Commands.Network.Models;
+    using System.Management.Automation;
+    using MNM = Microsoft.Azure.Management.Network.Models;
+    using System.Linq;
+
+    public class AzureRMExpressRouteCrossConnectionPeeringBase : NetworkBaseCmdlet
+    {
+        public const string IPv4 = "IPv4";
+        public const string IPv6 = "IPv6";
```

Comment:
We should look at moving these definitions to [NetworkBaseCmdlet](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Network/Commands.Network/Common/NetworkBaseCmdlet.cs)

I see that its defined/used at least in one more place

Reply:
Done